Documentation/rendering.md

```diff
@@ -78,10 +79,29 @@ The viewport shows the rendered model, streaming it from the ARR service. The ca
 Entities can be selected by clicking on them on the viewport. When you select an entity from the viewport, the corresponding entity gets selected and highlighted in the scene tree, and the other way around.
 Double-clicking on an entity will move the camera to frame the entity in the viewport.
 
-### Material list
+### Scene tree panel
+
+This panel shows a tree view with the hierarchy of all the entities in the loaded model. You can expand every node, and select them.
+
+### Material panel
+
+This panel shows on the top a list of the materials in the selected entities, or, if there's no selected entity, the full list of materials. Materials can be selected with mouse click.
+
+When a material is selected, the lower area shows all of its editable properties. You can change properties (colors, parameters, flags) and see the results immediately reflected in the rendered model. There is no support for editing the material textures. For more information on the supported ARR materials, see [Materials](https://docs.microsoft.com/azure/remote-rendering/concepts/materials)
+
+### Statistics panel
+
+![Statistics panel](media/statspanel.png)
+
+At the bottom of the windows you have access to the panel for the statistics. Here you can start and stop the stats collection and visualize the data of the current collection.
```

Comment:
Nit: For some reason, "panel for the statistics" doesn't sound quite right to me, possibly because "the statistics" would need to be introduced first. It sounds OK if you just say "to the statistics panel".